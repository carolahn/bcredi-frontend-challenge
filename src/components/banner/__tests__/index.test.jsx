import React from "react";
import renderer from "react-test-renderer";

import Banner from "../index";

describe("Render Test", () => {
  it("renders correctly Banner", () => {
    const tree = renderer.create(<Banner />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
