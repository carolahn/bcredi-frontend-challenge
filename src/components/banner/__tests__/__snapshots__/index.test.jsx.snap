// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders correctly Banner 1`] = `
<div
  className="sc-bdfBwQ hkoKkK banner"
>
  <div
    className="banner__text"
  >
    <p
      className="banner__quote"
    >
      "Obtive crédito para capital de giro. O processo foi bem sucedido e tudo que foi abordado, foi cumprido."
    </p>
    <p
      className="banner__name"
    >
      Camila Bragança,
    </p>
    <p
      className="banner__company"
    >
      Sideral Tecnologia
    </p>
  </div>
</div>
`;
